build-system/tasks/update-packages.js

```diff
@@ -104,10 +104,10 @@ function patchIntersectionObserver() {
 function patchResizeObserver() {
   // Copies intersection-observer into a new file that has an export.
   const patchedName =
-    'node_modules/resize-observer-polyfill/resize-observer.install.js';
+    'node_modules/resize-observer-polyfill/ResizeObserver.install.js';
   let file = fs
     .readFileSync(
-      'node_modules/resize-observer-polyfill/dist/ResizeObserver.global.js'
```

Comment:
From the docs:
> Note: global version of the polyfill (dist/ResizeObserver.global) is deprecated and will be removed in the next major release.

Reply:
Yep. I noticed it after the fact. So renamed it here.